openwisp_network_topology/integrations/device/tests.py

```diff
@@ -269,6 +269,7 @@ def test_monitoring_integration(self, *args):
                         'openwisp_monitoring.device',
                         'openwisp_monitoring.check',
                         'openwisp_controller.connection',
+                        'openwisp_notifications',
```

Comment:
this test was failing without this line

Reply:
please remove this and the other thing related to markdown, if there's an issue with this part it must be resolved in another patch, not in this one